README.md

```diff
@@ -0,0 +1,320 @@
+
+# Cat API #
+
+# A REST API using ExpressJS#
+
+A data repository to manage information about cats and breeds via the HTTP request-response cycle.
+
+# Getting Started #
+
+Run 'npm start' on Terminal.
+
+Access via web browser at ???
+
+## Motivation
+
+To create an API with full CRUD functionality to put knowledge of **Javascript** and the **ExpressJS** framework into practice.  
+It provided the opportunity to use the **mocha** framework, **jest** and **supertest** library for writing unit and integration tests. 
+
+It helped me to develop my **github** workflow for better management of *branches* and *pull requests*, to enable separation of concerns for easier review and working.    
+
+## Code Style
+
+This project follows standard JS codestyle and *contributions should be validated locally with ESLint*.
+
+## Tech/Framework Used
+
+- [ExpressJS](https://expressjs.com/) 
+- [Postman API development tool](https://www.postman.com/)
+- [Jest testing framework](https://jestjs.io/), [Mocha testing framework] (https://mochajs.org/) & [Supertest Testing Library](https://www.npmjs.com/package/supertest)
+- This project was bootstrapped with ???
+
+## Features
+
+The application consists of two repository classes - Cat and Breed object classes, respectively. 
+In addition to this, there are two 'database querying' Express Routers to handle Cat and Breed routes, respectively. 
+A seperate module consolidates simple functions for use across classes. All modules are then imported into the main server file.  
+
+The API codebase is structured using repository pattern, minimising code repetition and upholding separation of concerns. 
+A consistence interface rationalises expectations for input/outputs for middleware and JS functions.  
+These elements support persistence ignorance, easing future implementation of a third party database. 
+
+The application includes a series of unit and integration tests, strengthened by the process of Test Driven Development.
+Use of **Continuous Integration** allows for automated testing. 
+
+It increased familiarity with the processes of managing data formats, for example, parsing JSON objects, curl and url encoded data. 
+
+## Code Walkthrough 
```

Comment:
it'd be good to link to the file on Github's URL when giving a walkthrough. This is nice and comprehensive otherwise 👍